evennia/server/evennia_launcher.py

```diff
@@ -2278,8 +2281,11 @@ def main():
         # pass on to the core django manager - re-parse the entire input line
         # but keep 'evennia' as the name instead of django-admin. This is
         # an exit condition.
+
         sys.argv[0] = re.sub(r"(-script\.pyw?|\.exe)?$", "", sys.argv[0])
-        sys.exit(execute_from_command_line(sys.argv))
+        # sys.exit(execute_from_command_line(sys.argv))
+        django.core.management.call_command(*([option] + unknown_args))
```

Comment:
This `execute_from_command_line()` call was added as a result of a lot of bug fixing. The `call_command` simply does not act quite the same way as the draw command line call ... The difference is subtle and I don't recall exactly why at this point but I suspect this change will have unexpected side effects/regressions.

Reply:
The reason I changed it to use `call_command()` is because the plugins need to be able to modify the settings before Django begins its operations. if a whole new process is spawned to run the Django management then it will run using the settings specified in DJANGO_SETTINGS_MODULE environmental variable and will not include the settings from the plugins.

The only way around this that I'm aware of would be the previous method of plugins I used - having the `mygame/server/conf/settings.py` include some code which triggers the plugin initializing inline with importing the module. Perhaps at the very bottom.